t/07-status.t

```diff
@@ -0,0 +1,203 @@
+# vim:set ft= ts=4 sw=4 et:
+
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+repeat_each(2);
+
+plan tests => repeat_each() * blocks() * 5;
+
+my $pwd = cwd();
+
+our $MainConfig = qq{
+    lmdb_environment_path html/test.mdb;
+    lmdb_max_databases 3;
+    lmdb_map_size 5m;
+};
+
+our $MainConfig2 = qq{
+    lmdb_environment_path html/test.mdb;
+    lmdb_max_databases 3;
+    lmdb_map_size 10m;
+};
+
+our $HttpConfig = qq{
+    lua_package_path "$pwd/lib/?.lua;;";
+};
+
+no_long_string();
+#no_diff();
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: simple get env_info with 5M LMDB Map
+--- http_config eval: $::HttpConfig
+--- main_config eval: $::MainConfig
+--- config
+    location = /t {
+        content_by_lua_block {
+            local l = require("resty.lmdb")
+            local info = l.get_env_info()
+            ngx.say(info["map_size"])
+            ngx.say(info["page_size"])
+            ngx.say(info["map_size"]/info["page_size"]==info["max_pages"])
```

Comment:
let's also test other fields, we can use `response_body_like` and test against a pcre expression